### DISCUSSION TITLE
Adding 2FA prompts for my password (but I signed in with OAuth)

### DISCUSSION THREAD

**Discussion by dinkydani** [paircodedev/sandbox#10] [Bug Report] [answered]
I clicked Enable on 2FA and it pops up with a password prompt even though I log in with OAuth (Github)

<img width="782" alt="image" src="https://user-images.githubusercontent.com/3687223/120090499-8ee91700-c135-11eb-8265-c30769b29d4f.png">

**Comment by Necmttn** [comment]
Oh early adopter problems 😅 you sign up before i even implement random password upon signup feature 🙃 i will dm you random password 😇

**Comment by alserembani94** [comment]
I also have the same issue. Tried to do 2FA, but I logged in using OAuth as well (Github)

**Comment by Necmttn** [comment, marked as answer]
Removed the password confirmation when you're setting the 2fa right now.  🎉  this should resolve the problem.  Thank you for feedback guys @dinkydani  @alserembani94

**Reply by alserembani94** [reply]
2FA still asking me to confirm my password for security. Can you look on it again?

**Comment by alserembani94** [comment]
I am still facing the same problem after a month.

I tried to add 2FA, but it stills prompts me to confirm with my password (though I am using OAuth - Github). I tried the other way by changing the password, but it seems that I didn't have the current password, so I cannot change my password in any way.

Can you advice on this again?
